main.py

```diff
@@ -1,10 +1,56 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+
+    with open("players.json", "r") as f:
+        data = json.load(f)
+
+    for name, data_info in data.items():
+
+        race_name = data_info["race"]["name"]
+        race_note = data_info["race"]["description"]
+        if not Race.objects.filter(name=race_name).exists():
+            Race.objects.get_or_create(
+                name=race_name,
+                description=race_note
+            )
```

Comment:
You don't need the check that the object exists with the get_or_create() method. Fix in all other similar cases